registry.json

```diff
@@ -1997,5 +1997,20 @@
     "telegram":"retrostakepool",
     "operatorName":"Beau",
     "github":"retropool"
+  },
+  {
+    "ticker": "SBLYR",
+    "poolId": "8264de3cdb1798dd8758e24cda5101184b44543e7c4421c7815f9ed8",
+    "social": {
+      "twitter": "https://twitter.com/sublayerio",
+      "facebook": "",
+      "reddit": "https://reddit.com/user/sublayerio",
+      "instagram": "",
+      "website": "https://pool.sublayer.io",
+      "youtube": "https://www.youtube.com/channel/UC5MjFaqqlAbTX5NtxbG2_6Q"
+    },
+    "telegram": "sblyr",
```

Comment:
You need to provide a Telegram username.  A group name is not accepted.  The SPA uses Telegram to gather votes from it's members and we use the username to let our members in the voting group.

Reply:
personal telegram handle is sublayerio